### PR TITLE
Post Exchange Phishing Alerts to Slack/Microsoft Teams

### PR DESCRIPTION
**Pull request by mhofert-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams.icon [hunk 1]
```diff
@@ -987,7 +987,7 @@
               "input": {
                 "channel_name": "change_me",
                 "message_content": "📨  Email Received: Beginning Analysis 📨<br>\n<b>From:</b> {{[6a008e1f-33e2-476d-b292-c98d5faeb1db].[email].[sender]}}<br>\n<b>Subject:</b> {{[6a008e1f-33e2-476d-b292-c98d5faeb1db].[email].[subject]}}\n",
-                "team_name": "Dream Team"
+                "team_name": "change_me"
```
Comment (tylerslijboom-r7) [inline workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams.icon:990]:
```suggestion
                "team_name": "change-me"
```
For consistencies sake...sorry

Comment (mrinehart-r7) [reply workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams.icon:990]: Negative, it should be `change_me` as per

```
Jon Schipp: 8:38 AM
@here For all the hardcoded inputs that a user needs to customize to run the workflow, the new text value is change_me
```

File: workflows/Parse_Phishing_Indicators_with_Microsoft_Exchange_from_Slack/Parse_Phishing_Indicators_with_Microsoft_Exchange_from_Slack.icon [hunk 2]
```diff
@@ -1083,7 +1083,7 @@
               "input": {
                 "attachments": [],
                 "message": "{{[bf712037-937a-4f6c-8817-22e1801e0bac].[content]}}",
-                "recipient": "#change_me",
+                "recipient": "change-me",
```
Comment (mrinehart-r7) [inline workflows/Parse_Phishing_Indicators_with_Microsoft_Exchange_from_Slack/Parse_Phishing_Indicators_with_Microsoft_Exchange_from_Slack.icon:1086]:
```suggestion
                "recipient": "change_me",
```

File: workflows/Parse_Phishing_Indicators_with_Microsoft_Exchange_from_Slack/Parse_Phishing_Indicators_with_Microsoft_Exchange_from_Slack.icon [hunk 3]
```diff
@@ -1157,7 +1157,7 @@
               "input": {
                 "attachments": [],
                 "message": "📨  Email Received: Beginning Analysis 📨\n\nFrom: {{[6a008e1f-33e2-476d-b292-c98d5faeb1db].[email].[sender]}}\nSubject: {{[6a008e1f-33e2-476d-b292-c98d5faeb1db].[email].[subject]}}",
-                "recipient": "#change_me",
+                "recipient": "change-me",
```
Comment (mrinehart-r7) [inline workflows/Parse_Phishing_Indicators_with_Microsoft_Exchange_from_Slack/Parse_Phishing_Indicators_with_Microsoft_Exchange_from_Slack.icon:1160]:
```suggestion
                "recipient": "change_me",
```

File: workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams.icon [hunk 4]
```diff
@@ -1259,9 +1259,9 @@
             "isDisabled": false,
             "parameters": {
               "input": {
-                "channel_name": "change_me",
+                "channel_name": "change-me",
```
Comment (mrinehart-r7) [inline workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams.icon:1262]:
```suggestion
                "channel_name": "change_me",
```

File: workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams.icon [hunk 5]
```diff
@@ -1259,9 +1259,9 @@
...
                 "message_content": "{{[bf712037-937a-4f6c-8817-22e1801e0bac].[content]}}",
-                "team_name": "Dream Team"
+                "team_name": "change-me"
```
Comment (mrinehart-r7) [inline workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams.icon:1264]:
```suggestion
                "team_name": "change_me"
```

File: workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/help.md [hunk 6]
```diff
@@ -43,6 +43,7 @@ _There is no troubleshooting information at this time_
 
 # Version History
 
+* 1.0.1 - Change starting Microsoft Teams input fields to match new standards
```
Comment (mrinehart-r7) [inline workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/help.md:46]:
```suggestion
* 1.0.1 - Change Microsoft Teams input fields to match new standards
```

**Pull request merged**